easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
wait... do you have a reference for this?

I know you can get the Intel MPI runtime for free, but also the full thing that allows you to build/link with/to it?

@DanielTosello?